packages/dev/s2-docs/src/IconSearchView.tsx

```diff
@@ -48,6 +49,119 @@ let handleCopyImport = (id: string) => {
   });
 };
 
+interface IconSearchViewProps {
+  filteredItems: typeof iconList
+}
+
+export function IconSearchView({filteredItems}: IconSearchViewProps) {
```

Comment:
the height of this seems to be off, I have two scroll bars

<img width="1089" height="617" alt="Image" src="https://github.com/user-attachments/assets/610b710f-4a0f-4250-b105-a467754ffd0e" />